sdk/core/abort-controller/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+<!-- dev-tool snippets ignore -->
```

Comment:
should we just exclulde CHANGELOG.md from the files to process?

Reply:
There might be some cases where we want it, I didn't want to exclude them just yet.  If I see more of this, I will do so.